multiqc/modules/jellyfish/jellyfish.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse results from jellyfish  """
+
+from __future__ import print_function
+
+from collections import OrderedDict
+import logging
+from multiqc import config
+from multiqc.plots import linegraph, bargraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='jellyfish', anchor='jellyfish',
+        href="http://www.cbcb.umd.edu/software/jellyfish/",
+        info="JELLYFISH is a tool for fast, memory-efficient counting of k-mers in DNA.")
+
+        self.jellyfish_data_all = dict()
+        for f in self.find_log_files('jellyfish', filehandles=True):
+            self.parse_jellyfish_data(f)
+        
+        #clean histogram to prevent long flat tail
+        self.clean_jellyfish_data()
+        
+        if len(self.jellyfish_data) == 0:
+            log.debug("Could not find any data in {}".format(config.analysis_dir))
+            raise UserWarning
+            
+        log.info("Found {} reports".format(len(self.jellyfish_data)))
+        
+        self.intro += """Jellyfish k-mer plots for library complexity. 
+        A possible way to assess the complexity of a
+        library even in absence of a reference sequence is to
+        look at the kmer profile of the reads. The idea is to
+        count all the kmers (i.e., sequence of length k) that occur
+        in the reads. In this way it is possible to know how many
+        kmers occur 1,2,..., N times and represent this as a
+        plot. This plot tell us for each x, how many k-mers
+        (y-axis) are present in the dataset in exactly x-copies.
+        In an ideal world (no errors in sequencing, no bias, no
+        repeated regions) this plot should be as close as
+        possible to a gaussian distribution. In reality we will
+        always see a peak for x=1 (i.e., the errors) and another
+        peak close to the expected coverage. If the genome is
+        highly heterozygous a second peak at half of the coverage
+        can be expected."""
+        
+        
+        self.sections = list()
```

Comment:
The v1.0 release introduced a new syntax for this. See the [docs FAQ](http://multiqc.info/docs/#updating-for-compatibility) for more info. In short, could you please change this to:

```python
self.add_section(
  name = 'Jellyfish k-mers',
  anchor = 'jellyfish_kmer_plot',
  description = 'Jellyfish plot for k-mers with coverage between 0 and {}'.format(self.jellyfish_max_x),
  plot = self.frequencies_plot()
)
```

Note that I think `name` is used in the side navigation if there is ever a second section for this module, so should be relatively short if possible.